sw/device/tests/sha256_test.c

```diff
@@ -19,27 +18,22 @@ static const uint32_t kExpectedDigest[8] = {0xdc96c23d, 0xaf36e268, 0xcb68ff71,
                                             0xe92f76e2, 0xb8a8379d, 0x426dc745,
                                             0x19f5cff7, 0x4ec9c6d6};
 
-int main(int argc, char **argv) {
-  uart_init(kUartBaudrate);
-  uart_send_str("Running SHA256 test\r\n");
+bool test_main(void) {
+  LOG_INFO("Running SHA256 test");
```

Comment:
Hmm... wonder if there's some way to make this common code in testlib? (Not relevant to this CL, just an idle thought.)

Reply:
Agree - this can be moved to the common `main()` as well.